.github/workflows/test-and-publish-nightly.yml

```diff
@@ -35,6 +35,10 @@ jobs:
               display: linux-x64
               name: ubuntu-latest
             image: public.ecr.aws/async-profiler/asprof-builder-x86:latest
+          - runson:
+              display: amazonlinux2
+              name: ubuntu-latest
+            image: public.ecr.aws/async-profiler/asprof-builder-amazonlinux:al2
```

Comment:
"al" both in the name and in the version looks a bit clumsy.
What about `asprof-builder-al2:latest` or `asprof-builder-amazonlinux:2` ?

Reply:
`asprof-builder-amazonlinux:2` :+1: